xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -324,6 +325,7 @@ class CAddonEpgTag : public EPG_TAG
   const std::string m_writer;
   const std::string m_IMDBNumber;
   const std::string m_episodeName;
+  int m_episodePartNumber;
```

Comment:
Please remove this line. A member is only needed for `const char*`API struct members.

Reply:
Sure